src/commands/e2e/E2E.ts

```diff
@@ -224,6 +230,22 @@ export class E2E implements ICommand {
         return true;
     }
 
+    private filterPluginsByHasSpecifiedTests(): void {
+        this.pluginsToBeTested = this.pluginsToBeTested
+            .filter((plugin) => {
+                        const pathToSpecFiles = path.join(this.workingDir, plugin, 'assets', 'e2e', 'specs');
+                        if (fs.existsSync(pathToSpecFiles)) {
+                            if (glob.sync(path.join(pathToSpecFiles, '**', '*.spec.ts')).length > 0) {
+                                Global.isVerbose() && console.warn(`Plugin ${plugin} has specified .spec.ts test`);
+                                return true;
+                            }
+                        }
+                        Global.isVerbose() && console.warn(`Plugin ${plugin} does not have an E2E specs folder - please provide the folder and specify a Test first`);
```

Comment:
I don't think we should print a warning here. Also the warning would be printed even if the directory exists but there are no spec files...